src/main/java/com/jvm_bloggers/core/data_fetching/blogs/BloggersDataFetcher.java

```diff
@@ -57,12 +88,15 @@ private void refreshBloggersDataFor(Optional<URL> blogsDataUrl, BlogType blogTyp
                 BloggersData bloggers = mapper.readValue(blogsDataUrl.get(), BloggersData.class);
                 bloggers.getBloggers().stream().forEach(it -> it.setBlogType(blogType));
                 bloggersDataUpdater.updateData(bloggers);
-            } catch (IOException exception) {
+            } catch (Exception exception) {
```

Comment:
Why we are widening the type of the caught exception here? Isn't it a bad code smell to catch general `Exception`?